src/store/index.js

```diff
@@ -81,6 +103,71 @@ const actions = {
     }).catch((error) => {
       console.log(`エラー発生：${error}`)
     })
+  },
+  async updateWallet ({commit}, {destinationId, destinationWallet, currentId, currentWallet}) {
+    // DBの金額更新
+    const destinationDoc = db.collection('users').doc(destinationId)
+    const currentWalletDoc = db.collection('users').doc(currentId)
+    return db.runTransaction(async (transaction) => {
+      // 送金される側の更新
+      transaction.update(destinationDoc, { wallet: destinationWallet })
+      console.log('送信先の残高更新に成功しました')
+      // 送金する側の更新
+      transaction.update(currentWalletDoc, { wallet: currentWallet })
+      console.log('ログインユーザの残高更新に成功しました')
+    }).then(() => {
+      console.log('Transaction successfully committed!')
+      // DBの金額更新後、userListを更新
+      db.collection('users').get().then((query) => {
+        console.log('ユーザリストの参照に成功しました')
+        const buff = []
+        query.forEach((doc) => {
+          const data = doc.data()
+          buff.push([doc.id, data.name, data.email, data.wallet])
+        })
+        commit('setUserList', buff)
+        const createUserArray = buff.filter(doc => doc[1] === state.username)
+        const createUser = createUserArray[0]
+        commit('setWallet', createUser[3])
+        // モーダルウィンドウ非表示
+        commit('setShowSendModal', false)
+      }).catch(error => {
+        console.log(`ユーザリストの参照に失敗しました：${error}`)
+      })
+    }).catch((error) => {
+      console.log(`Transaction failed: ${error}`)
+    })
+  },
+  signCheck ({commit}) {
+    firebase.auth().onAuthStateChanged(user => {
+      if (user) {
+        console.log('login')
+      } else {
+        console.log('logout')
+      }
+    })
+  },
+  async getUserLists ({commit}) {
+    db.collection('users').get().then((query) => {
```

Comment:
以下のコードですとこちらの環境でも動くことが確認できていますが、いかがでしょうか？
```suggestion
  getUserLists ({commit}) {
    return db.collection('users').get().then((query) => {
```